docs/pages/components/Notice.mdx

```diff
@@ -0,0 +1,192 @@
+or anywhere else in the DOM without needing to be absolutely/statically positioned.
+
+In addition, the `Notice` component has only limited dynamic functionality built in, allowing developers to
+have full control over the underlying element via its `forwardRef`, while still having useful base styling..
+
+```jsx example
+<Notice type="info" title='Information'>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem.
+  Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.
+</Notice>
+
+<Notice type="caution" title="Caution">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem.
+  Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.
+</Notice>
+```
+
+### Notice With Icon
+
+```jsx example
+<Notice
+  type="info"
+  title={
+    <h3
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      Information
+    </h3>
+  }
+>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+
+<Notice
+  type="caution"
+  title={
+    <h3
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      Caution
+    </h3>
+  }
+>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Different icon
+
+<Note>
+  The `icon` prop takes a string that corresponds to a [Cauldron
+  Icon](/components/icon). If you would like to use a custom icon, you can
+  compose the contents of the `Notice` component (see
+  <a href="/components/Notice#notice-with-composed-contents">example</a>).
+</Note>
+
+```jsx example
+<Notice type="caution" title="Dynamo!" icon="bolt">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Composed Contents
+
+```jsx example
+<Notice>
+  <div className="Notice__title">
+    <Icon type="star" />
+    <h2
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      You are a star!
+    </h2>
+  </div>
+  <p style={{ margin: 'var(--space-half) 0 0 var(--space-half)' }}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at
+    porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus
+    arcu condimentum sed.
+  </p>
+</Notice>
+```
```

Comment:
What's this?

Reply:
It's a usage where just the base styles are being used and the component's contents are composed. People don't have to do this if the built in props get them where they want to go, but figured it would be nice to have that flexibility.

Reply:
I'm not sure this is a pattern we want to encourage in documentation vs. something like `Panel`. `Panel` is intended to be a generic container component, where this should have a more restrictive interface.

Reply:
That's fair/reasonable and I'll go ahead and remove it.